_posts/history/2022-02-06-a-brief-history-of-usrse.md

```diff
@@ -0,0 +1,372 @@
+
+![US-RSE growth February 2022]({{ site.baseurl }}/assets/posts/history/2022-02-brief-history-usrse/growth.png "Growth of US-RSE has been consistent!")
+
+This automated graphic was added in May 2019 by Vanessa Sochat [[ref](https://github.com/USRSE/usrse.github.io/pull/85)], who also created and added the community map [[ref](https://github.com/USRSE/usrse.github.io/pull/25)] that is now maintained on its own site [[ref](https://us-rse.org/usrse-map/)].
+The chart shows change over time, while the map shows a single snapshot in time of membership based on those who choose to report their city and state.
+In a larger sense, what these small initiatives capture is the general excitement that was held by the community - not just to find others like themselves, but to feel like they can be a part of a larger community and movement.
+And because many community members are software engineers, it was fairly natural to have things like automation, continuous integration, and new web interfaces added to make this possible.
+
+In mid-2019, Ian Cosden, Jeff Carver, and Dan Katz, in an effort to seek funding to grow the organization, began conversations with The Alfred P. Sloan Foundation.
+The three wrote a proposal for a dedicated, in-person, [US-RSE community building workshop](https://us-rse.org/first-community-workshop/) designed to bring together community members to plan the best path forward for US-RSE.
+The [grant](https://sloan.org/grant-detail/9227) was funded and a workshop was scheduled for April 2020 in Princeton, NJ.
+Unfortunately, it was bad timing, and COVID-19 forced the repeated postponement of the workshop. 
+
+
+### Community Voices
+
+In July of 2019, after joining US-RSE in May, community member Vanessa Sochat was frustrated about the message being sent about what constituted an RSE.
+Her first effort to send a message of a diversity of roles was to make a video, "The Story of the Research Software Engineer" [[ref](https://www.youtube.com/watch?v=trAfA9VWLTQ&t=3s)], however any video can only have limited impact.
+She woke up one morning in September, and knew that she needed to start a podcast, and lift up research software engineers from all around the community to tell their stories directly.
+This became the RSE Stories podcast [[ref](https://us-rse.org/rse-stories)] which eventually brought on a host from the UK, and inspired the creation of other RSE-run podcasts [[ref](https://codeforthought.buzzsprout.com/)].
+
+
+### Community Events
+
+TBD: workshops, conference events, etc. that were organized – definitely the two actual US-RSE events (don't remember what they were called) – virtual mini-conferences
+
+
+### Community Calls and Newsletters
+
+TBD - should be put in as a major community effort - first community call and first newsletter were both in July 2019
```

Comment:
another TBD that needs to be fleshed out

Reply:
We can also just remove the section.

Reply:
we could remove this sentence, since the community calls are discussed here - but maybe we should add something about the newsletters?

Reply:
Maybe "US-RSE started creating and distributing [newsletters](https://us-rse.org/newsletters/) in July 2019, led by Sandra Gesing, and has generally tried to issue them every two months, as one of our means of sharing information among members."

Reply:
Done!